tools/python3/phloping.py

```
@@ -143,9 +153,16 @@ def normalised_times_for_L(self, ilvl):
         ).reshape(times.shape[0])
 
 
-def file_parser(run, rank, times_filepath):
-    supe = st.file_parser(times_filepath)
-    return ScopeTimerFile(supe.id_keys, supe.roots, run, str(rank))
+def file_parser(run, times_filepath):
+    supe = phst.file_parser(times_filepath)
+    return ScopeTimerFile(supe.id_keys, supe.roots, run)
+
+
+def make_scope_timer_file_from(input):
```

Ruff (0.15.14)

[error] Function argument `input` is shadowing a Python builtin

(A002)